bench/README.md

```diff
@@ -18,40 +28,214 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
+* Copy local accounts keys to folder `~/.dcl/keyring-test`:
+    ```bash
+    cp ./.localnet/node0/keyring-test/* ~/.dcl/keyring-test
+    ```
+* Enter to `bench` folder
+    ```bash
+    cd bench
+    ```
+* Build `docker-compose` file
+    ```bash
+    docker-compose build
+    ```
+* Open the docker-compose.yml:
+    * update fields `DCLD_VERSION`, `DCLD_NODE`, `DCLD_CHAIN_ID` for `args`.<br>
+        e.g.:
+        ```yml
+        DCLD_VERSION: v0.11.0
+        DCLD_NODE: tcp://host.docker.internal:26657
+        DCLD_CHAIN_ID: dclchain
+        ```
+    * update field `extra_hosts`.<br>
+        e.g.:
+        ```yml
+        extra_hosts:
+          - "host.docker.internal:host-gateway"
+        ```
+        `<host.docker.internal:host-gateway>` - need to connecting to localhost from locust container.
+
+## Run
+
+### (Optional) Launch Prometheus
+
+```bash
+prometheus --config.file=bench/prometheus.yml
+```
+
+And open `http://localhost:9090/` to query and monitor the server-side metrics.
+
+## Headless
+
+To run write load tests
+
+* Open the docker-compose.yml and update field `command` for `master` and `worker`.  
+    e.g.: for `master`
+    ```yml
+    command: -f /dcl/bench/locustfile.py --headless WriteModelLoadTest --master -H http://master:8089 
```

Comment:
You could refactor `command` field in docker-compose.yml in the following way:
```yaml
command:
  - "-f"
  - "/dcl/bench/locustfile.py"
  - "--headless"
  - "--master"
  - "-H"
  - "http://master:8089"
  - "WriteModelLoadTest"
```
This could simplify writing instructions

Reply:
Done! ✅